es/refresher-linear-algebra.md

```diff
@@ -126,49 +126,49 @@
 
 **22. Transpose ― The transpose of a matrix A∈Rm×n, noted AT, is such that its entries are flipped:**
 
-&#10230; Transpuesta ― La transpuesta de la matriz A∈Rm×n, con notación AT, es tal que sus entradas son volteadas:
+&#10230; Transpuesta ― La transpuesta de una matriz A∈Rm×n, denotada AT, está dada por el siguiente cambio de posición de sus entradas:
```

Comment:
(Optional) Can we find a translation for "is such that its entries are flipped" that does not change the original text too much? Maybe like _"es tal que sus entradas se intercambian de la forma"_: